components/example-notebook-servers/README.md

```diff
@@ -0,0 +1,139 @@
+# Example Notebook Servers
+
+**DISCLAIMER:** The Notebooks Working Group provides these Dockerfiles and
+their images as examples only, they are not tested with our CI/CD pipelines
+and are not certified to work in every situation. As such, issues related to
+these images will be dealt with in a best efforts approach. The Notebooks
+Working Group will work on officially supporting some of these Notebook
+Server images in the near future. If you do encounter a problem in one of
+these images, contributions and issue reports are greatly appreciated.
+
+## Introduction
+
+This directory contains Dockerfiles that are intended to show users how to
+create images for various types of Notebook Servers that can run on Kubeflow.
+There is a common base images that starts from Ubuntu 20.04, adds some
+dependencies and common utilities from APT, installs `kubectl`, creates the
+user account for the downstream images and adds the
+[S6-overlay](https://github.com/just-containers/s6-overlay) for the init system.
```

Comment:
Covert this to a bullet list? E.g.:

A common base image provides the following layers:

- Ubuntu 20.04
- `kubectl`
- User account (name?)
- init-system

Reply:
I didn't want to go into detail about the images themselves here (leave that to the READMEs for each image), but it does make sense to add the base image specification here. 

The user account is still `jovyan` at the moment (for compatibility reasons), for 1.4 this should probably be changed to `kubeflow`.